src/frontend/web_application/webpack/config.js

```diff
@@ -51,7 +51,7 @@ const configureStylesheet = (config, filename = 'client_[name]', relativePath =
     },
     {
       test: /\.scss$/,
-      loader: extractTextPlugin.extract('style-loader', 'css-loader?sourceMap!sass-loader'),
+      loader: extractTextPlugin.extract('style-loader', 'css-loader?sourceMap!postcss-loader?sourceMap!sass-loader'),
```

Comment:
Bear in mind that if `css-loader` `minimize` option is on, then `autoprefixer` will be run twice, because `minimize` uses `cssnano`. And I had a hard time making the cssnano version use our browsers config.

`autoprefixer` add prefixes, but also remove them if not needed. So you get the idea...